src/store/Controller.h

```diff
@@ -39,10 +39,22 @@ class Controller: public Storage
     virtual void sync() override;
     virtual void maintain() override;
     virtual void markForUnlink(StoreEntry &) override;
+    virtual void unlinkByKeyIfFound(const cache_key *key) override;
```

Comment:
please follow the updated coding style in this file, not mentioning parameter names unless they are needed to document the parameter better than type or method name can. StoreEntry and cache_key are already descriptive enough types to convey the necessary detail.

Please also consider using the new style in the other new function and methods. I'm not asking for those others as a review item there since it is polish and they are situated in places not using the new style yet.